app/components/osf-footer/component.ts

```diff
@@ -0,0 +1,12 @@
+import Component from '@ember/component';
+import config from 'ember-get-config';
+import { serviceLinks } from 'ember-osf-web/const/service-links';
+
+export default class OsfFooter extends Component {
+    serviceLinks = serviceLinks;
+    supportEmail: string = '';
+    constructor() {
+        super(...arguments);
+        this.supportEmail = config.support.supportEmail;
```

Comment:
Minor/FYI: Assigning to a property in the constructor is exactly the same as in the class body, so this would be equivalent:
```ts
export default class OsfFooter extends Component {
    serviceLinks = serviceLinks;
    supportEmail: string = config.support.supportEmail;
}
```
I'd usually only do it in the constructor if there's more logic to it, but that's down to a matter of taste.